src/components/Breadcrumbs/README.md

```diff
@@ -1,43 +1,345 @@
-## Breadcrumbs
+<!--GITHUB_BLOCK-->
 
-Breadcrumbs component. Can collapse breadcrumbs that cause overflow.
+# Breadcrumbs
 
-### PropTypes
+<!--/GITHUB_BLOCK-->
 
-| Name                     | Type       | Required | Default | Description                                                                                                                  |
-| :----------------------- | :--------- | :------: | :------ | :--------------------------------------------------------------------------------------------------------------------------- |
-| items                    | `Array`    |    ✓     |         | Breadcrumb items array `BreadcrumbsItem[]`                                                                                   |
-| className                | `String`   |          |         | CSS class name of root element                                                                                               |
-| renderRootContent        | `Function` |          |         | Custom render function of first item `(item: BreadcrumbsItem, isCurrent: boolean) => React.ReactNode;`)                      |
-| renderItemContent        | `Function` |          |         | Custom render function of N+1 item `(item: BreadcrumbsItem, isCurrent: boolean, isPrevCurrent: boolean) => React.ReactNode;` |
-| renderItemDivider        | `Function` |          |         | Custom render function of items separator `() => React.ReactNode;`                                                           |
-| lastDisplayedItemsCount  | `Enum`     |    ✓     |         | Number of items to display after items collapse control: `LastDisplayedItemsCount`                                           |
-| firstDisplayedItemsCount | `Enum`     |    ✓     |         | Number of items to display before items collapse control: `FirstDisplayedItemsCount`                                         |
-| popupStyle               | `String`   |          |         | Style of collapsed items popup `staircase`                                                                                   |
+```tsx
+import {Breadcrumbs} from '@gravity-ui/uikit';
+```
+
+`Breadcrumbs` can collapse breadcrumbs that cause overflow.
```

Comment:
I think we need a little bit more suitable description. [Smth like this](https://mui.com/material-ui/react-breadcrumbs/)